edge-apps/menu-board/screenly.yml

```diff
@@ -0,0 +1,711 @@
+---
+syntax: manifest_v1
+description: A dynamic menu board that can display up to 25 menu items
+author: Screenly, Inc
+entrypoint:
+  type: file
+ready_signal: true
+settings:
+  accent_color:
+    type: string
+    default_value: rgba(255, 255, 255, 0.95)
+    title: Accent Color
+    optional: true
+    help_text: Color used for titles and prices (default is white with a subtle purple tint)
+  background_image:
+    type: string
+    default_value: assets/pizza.png
+    title: Background Image URL
+    optional: true
+    help_text: URL to a background image. For best results, use a dark/subtle image.
+  item_01_description:
+    type: string
+    default_value: San Marzano tomatoes, fresh mozzarella, basil, extra virgin olive oil
+    title: Item 1 - Description
+    optional: true
+    help_text: Description of menu item 1
+  item_01_labels:
+    type: string
+    default_value: vegetarian
+    title: Item 1 - Labels
+    optional: true
+    help_text: Comma-separated list of labels (e.g. vegetarian, spicy, gluten-free)
+  item_01_name:
+    type: string
+    default_value: Classic Margherita
+    title: Item 1 - Name
+    optional: true
+    help_text: Name of menu item 1
+  item_01_price:
+    type: string
+    default_value: '13.99'
+    title: Item 1 - Price
+    optional: true
+    help_text:
+      properties:
+        help_text: Price of menu item 1
+        type: number
+      schema_version: 1
+  item_02_description:
+    type: string
+    default_value: Double pepperoni, mozzarella, parmesan, homemade tomato sauce, oregano
+    title: Item 2 - Description
+    optional: true
+    help_text: Description of menu item 2
+  item_02_labels:
+    type: string
+    default_value: spicy
+    title: Item 2 - Labels
+    optional: true
+    help_text: Comma-separated list of labels (e.g. vegetarian, spicy, gluten-free)
+  item_02_name:
+    type: string
+    default_value: Pepperoni Supreme
+    title: Item 2 - Name
+    optional: true
+    help_text: Name of menu item 2
+  item_02_price:
+    type: string
+    default_value: '15.99'
+    title: Item 2 - Price
+    optional: true
+    help_text:
+      properties:
+        help_text: Price of menu item 2
+        type: number
+      schema_version: 1
+  item_03_description:
+    type: string
+    default_value: Mozzarella, gorgonzola, parmesan, fontina, fresh basil, garlic olive oil
+    title: Item 3 - Description
+    optional: true
+    help_text: Description of menu item 3
+  item_03_labels:
+    type: string
+    default_value: vegetarian
+    title: Item 3 - Labels
+    optional: true
+    help_text: Comma-separated list of labels (e.g. vegetarian, spicy, gluten-free)
+  item_03_name:
+    type: string
+    default_value: Four Cheese
+    title: Item 3 - Name
+    optional: true
+    help_text: Name of menu item 3
+  item_03_price:
+    type: string
+    default_value: '16.99'
+    title: Item 3 - Price
+    optional: true
```

Comment:
Would currency be a part of this or a separate field if required?

Reply:
Thanks, @iamplugged. Adding a separate field for the currency is better.